kernel/src/process/process/timer_manager.rs

```diff
@@ -31,6 +31,16 @@ use crate::{
 /// invoke the callbacks of expired timers which are based on the updated
 /// CPU clock.
 fn update_cpu_time() {
+    let is_kernel_interrupted = {
+        let interrupt_level = InterruptLevel::current();
+        let InterruptLevel::L1(cpu_priv_at_irq) = interrupt_level else {
+            // We are at the interrupt level 2.
+            // This means that bottom half of IRQ handling is interrupted.
+            // We should not count this time slice on the head of the current task.
+            return;
+        };
+        cpu_priv_at_irq == PrivilegeLevel::Kernel
+    };
```

Comment:
Insert a blank line below? This is not related to the thread/process stuffs.

Reply:
I tried. But the CI failed because `cargot fmt --check` complained the blank line.

```rust
cargo fmt --check
Diff in /Users/tatetian/Workspace/asterinas/kernel/src/process/process/timer_manager.rs:41:
         };
         cpu_priv_at_irq == PrivilegeLevel::Kernel
     };
-    
+
     let Some(current_thread) = Thread::current() else {
         return;
     };
```

It surprises me that cargo fmt does not like the blank line. But I don't think we need to fight against cargo fmt.

Reply:
<img width="499" height="448" alt="image" src="https://github.com/user-attachments/assets/3f43d940-bd5b-4e05-b397-0980aa785ace" />

Reply:
> It is due to you wrongly insert some space...

I see... In an attempt to make cargo fmt happy, I added some comments in addition a new blank line...

Reply:
`make format` automatically removes these trailing spaces, so it's a good habit to always use it before committing or pushing :(

Reply:
> Its default behavior is to invoke language-specific formatter on every file save

Many IDEs should be able to do that, but some may require special configurations. I'm not sure.

> Do you use VSCode as your daily IDE?

NeoVim users here. Zed does sound promising, and I checked it before. But I think I'm stuck using old-fashioned tools. I don't see any significant problems with them, so I don't want to switch.